bitnami/harbor/README.md

```diff
@@ -347,3 +347,36 @@ This chart includes a `values-production.yaml` file where you can find some para
 It is strongly recommended to use immutable tags in a production environment. This ensures your deployment does not change automatically if the same tag is updated with a different image.
 
 Bitnami will release a new chart updating its containers if a new version of the main container, significant changes, or critical vulnerabilities exist.
+
+## Upgrade
+
+## 2.0.0
+
+In this version, this **chart depends on PostgreSQL 11 instead of PostgreSQL 10**. You can find the main difference and notable changes in the following links: [https://www.postgresql.org/about/news/1894/](https://www.postgresql.org/about/news/1894/) and [https://www.postgresql.org/about/featurematrix/](https://www.postgresql.org/about/featurematrix/).
```

Comment:
I would also mention that Redis goes from version 4.0 to 5.0.

Reply:
Good point! Updated